map/generate-offline-map-overrides/README.md

```diff
@@ -1,35 +1,26 @@
-# Generate Offline Map (Overrides)
+# Generate offline map (overrides)
 
 Take a web map offline with additional options for each layer.
 
-For applications where you just need to take all layers offline, use the standard workflow (using only `GenerateOfflineMapParameters`). For a simple example of how you take a map offline, see the "Generate Offline Map" sample. For more fine-grained control over the data you are taking offline, use overrides to adjust the settings for each layer. Some example use cases for the overrides approach could be when you need to:
+![Image of generate offline map overrides](GenerateOfflineMapOverrides.png)
```

Comment:
Either describe what is actually depicted in the image or leave blank.

Reply:
@tschie just to let you know that these alt texts have been approved by the accessibility team at Esri: the samples team went through a big effort of adding them all in a standardised way. We thought also that describing what was in the image was the way to go but turns out for Esri guidelines that's not needed for this.